scripts/gen_js_protos.bash

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
Script for generating the JS proto files. It's gets pretty manual at the end because the hub case needs to be treated differently (requires billing protos as well).